src/Azure.Functions.ArtifactAssembler/Program.cs

```diff
@@ -14,8 +14,9 @@ static async Task<int> Main(string[] args)
                 // Check if an argument for zipping is passed
                 if (args.Length > 0 && args[0].Equals("zip", StringComparison.OrdinalIgnoreCase))
                 {
-                    var zipCliArtifacts = new CliArtifactZipper(currentWorkingDirectory);
-                    zipCliArtifacts.ZipCliArtifacts();
+                    var zipArtifacts = new ArtifactZipper(currentWorkingDirectory);
+                    zipArtifacts.ZipCliArtifacts();
+                    zipArtifacts.ZipVisualStudioArtifacts();
```

Comment:
Is the additional method call in line 19 change unrelated to the E2E test change?  Why do we need that for E2E test? Would it impact any other flow?

Reply:
no it's related. so initially we are zipping up the VS artifacts in the artifact consolidation step itself. Now we don't want to as we are running e2e tests on it and want to ensure it works. So after the test is run on the unzipped version, calling the zip param on the artifact assembler would also zip the VS artifacts